pkgs/top-level/all-packages.nix

```diff
@@ -184,6 +184,25 @@ with pkgs;
     autoconf = autoconf271;
   };
 
+  gnulibBootstrapHook = callPackage (
+    {
+      makeSetupHook,
+      gnulib,
+      autoconf,
+      automake,
+    }:
+    makeSetupHook {
+      name = "gnulib-bootstrap-hook";
+      propagatedBuildInputs = [
+        autoconf
+        automake
+      ];
+      substitutions = {
+        inherit gnulib;
+      };
+    } ../build-support/setup-hooks/gnulib-bootstrap.sh
+  ) { };
```

Comment:
I _think_ this can be in `pkgs/by-name/gn/gnulibBootstrapHook`. I'd prefer that.

Reply:
Hm, I’d at least expect pkgs/by-name to be case-insensitive (or forcing all to be lower-case) since that’s a reasonable behavior given that macOS and Windows use case-insensitive filesystems by default. I’m not sure if that’s the case though.

Reply:
We have a check in `nixpkgs-vet` that names can't differ by case -- `food` and `FooD` are rejected. macOS and Windows are case insensitive but case-_preserving_, so we can use the filesystem for determining attrset names as `pkgs/by-name` does.